packages/block-library/src/comment-template/edit.js

```diff
@@ -28,6 +33,60 @@ function CommentTemplateInnerBlocks() {
 	return <li { ...innerBlocksProps } />;
 }
 
+/**
+ * Component that renders a list of (nested) comments. It is called recursively
+ * in its own body.
+ *
+ * @param {Object} props                         Component props.
+ * @param {Array}  [props.blockProps]            - Props from parent's `useBlockProps()`.
+ * @param {Array}  [props.blockContexts]         - Array of comment objects.
+ * @param {Array}  [props.activeBlockContext]    - The block that is currently active.
+ * @param {Array}  [props.setActiveBlockContext] - The setter for activeBlockContext.
+ * @param {Array}  [props.blocks]                - Array of blocks returned from
+ *                                               getBlocks() in parent .
+ * @return {WPElement}                 		List of comments.
+ */
+const RenderComments = ( { blockContexts, ...props } ) => (
+	<ol { ...props.blockProps }>
+		{ blockContexts &&
+			blockContexts.map( ( blockContext ) => (
+				<BlockContextProvider
+					key={ blockContext.commentId }
+					value={ blockContext }
+				>
+					{ blockContext ===
+					( props.activeBlockContext || blockContexts[ 0 ] ) ? (
+						<>
+							<CommentTemplateInnerBlocks />
+							{ blockContext.children.length > 0 ? (
+								<RenderComments
```

Comment:
`RenderComments` should be rendered inside the `<li>` element so I guess it should be moved to `CommentTemplateInnerBlock`. In fact, it might be simpler to move all the code defined in the conditional `( props.activeBlockContext || blockContexts[ 0 ] ) ? (` to the same component.

Reply:
Done in a2c3983